### ISSUE TITLE
Structure of dataset

### ISSUE THREAD

**Issue by testerpce** [DVLP-CMATERJU/RectiNet#6]
I am having trouble understanding the directory structure. I have generated the images using the AGUN repository you have linked. However, I am just trying to understand how to use those images in the directory structure. Like which images go in the image folder of datagen ? And which is the label .npz file required.

**Comment by hmrishavbandy** [comment]
The generated images go to the image folder of datagen. The ground truth grid is saved as a pickle file. Since i find dealing with npz arrays easier, I used an npz array for saving the data.

**Comment by ARnnn** [comment]
> The generated images go to the image folder of datagen. The ground truth grid is saved as a pickle file. Since i find dealing with npz arrays easier, I used an npz array for saving the data.

It is the ground truth grid saved in .npz file?

**Comment by hmrishavbandy** [comment]
Yes. The ground truth grid is stored in the .npz file. Just change the manner of saving data from pkl to npz. Or you can make that change in the dataloader too. There was some issue with the pkl file...Something to do with floating point values iirc.

**Comment by ARnnn** [comment]
> Yes. The ground truth grid is stored in the .npz file. Just change the manner of saving data from pkl to npz. Or you can make that change in the dataloader too. There was some issue with the pkl file...Something to do with floating point values iirc.

![image](https://user-images.githubusercontent.com/52728076/109940130-b5e9a580-7d0c-11eb-9977-c27a50f43472.png)
When I save the grid truth in. pkl as. npz file and use the data set for training, this kind of error appears. How can I solve it

**Comment by ARnnn** [comment]
> Yes. The ground truth grid is stored in the .npz file. Just change the manner of saving data from pkl to npz. Or you can make that change in the dataloader too. There was some issue with the pkl file...Something to do with floating point values iirc.

When I save the grid truth in. pkl as. npz file and use the data set for training, this kind of error appears. How can I solve it
![image](https://user-images.githubusercontent.com/52728076/109943175-debf6a00-7d0f-11eb-9b11-eb4f42537730.png)

**Comment by hmrishavbandy** [comment]
Fixed. Apparently the wrong variable was being returned

**Comment by zbzzz** [comment]
How many images did you use as a data set at that time
